HerPortal/Views/Shared/_Layout.cshtml

```diff
@@ -54,15 +54,17 @@
     <a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>
 
     @{
-        List<HeaderNavigationViewModel> headerNavigationLinks = new List<HeaderNavigationViewModel>
+        var headerNavigationLinks = new List<HeaderNavigationViewModel>();
+
+        if (User.Identity?.IsAuthenticated is true)
         {
-            new()
+            headerNavigationLinks.Add(new HeaderNavigationViewModel
             {
                 Text = $"({User.GetEmailAddress()}) Sign Out",
```

Comment:
`User.GetEmailAddress()` fails when accessing anonymously (i.e. for the health check).

We could make it return `null`/a default value, but I thought it was safer not to leave it as it's used elsewhere.

Reply:
This looks good to me!